src/components/customizableAside.module.css

```diff
@@ -0,0 +1,27 @@
+/* customizableAside css */
+
+    .asideContainer {
```

Comment:
CSS properties are `kebab-case` not `camelCase`

`asideContainer` should be `aside-container` and so on.